gammapy/image/utils.py

```diff
@@ -121,11 +122,20 @@ def disk_correlate(image, radius, mode='constant'):
 
     Parameters
     ----------
-    TODO
+    image : ~numpy.ndarray
+        Image to be correlated.
+    radius : float
+        Disk radius in pixels.
+    mode : {'reflect','constant','nearest','mirror', 'wrap'}, optional
+	the mode parameter determines how the array borders are handled.
+        For 'constant' mode, values beyond borders are set to be cval.
+        Default is 'constant'.
 
     Returns
     -------
-    TODO
+    convolve : ndarray
```

Comment:
Please change
```
ndarray
```
to
```
`~numpy.ndarray`
```
everywhere, so that Sphinx generates a link to the numpy docs.

We could decide not to do it for ndarray because pretty much everyone should know what that is, but Python beginners might not and I think it's simplest to just always generate links to the classes we're talking about everywhere.